proof/infoflow/refine/ADT_IF_Refine.thy

```diff
@@ -304,9 +304,12 @@ lemma kernel_entry_if_corres:
       apply (rule corres_split)
          apply simp
          apply (rule threadset_corresT)
-            apply (erule arch_tcb_context_set_tcb_relation)
-           apply (clarsimp simp: tcb_cap_cases_def)
-          apply (rule allI[OF ball_tcb_cte_casesI]; clarsimp)
+               apply (erule arch_tcb_context_set_tcb_relation)
```

Comment:
was the indentation wrong here previously? it looks like `threadset_corresT` generates 6 subgoals whereas previously it generated 3?

Reply:
It was correct before. `threadSet_corresT` now includes some assumptions about the `tcbSchedPrev/Next` fields and `tcbQueued` because of the new `ready_queues_relation`.